src/Chart/index.tsx

```diff
@@ -15,6 +21,7 @@ export type Props = typeof ReactMinimalPieChart.defaultProps & {
   cx?: number;
   cy?: number;
   data: Data;
+  filterSegments: FilterSegmentsProp;
```

Comment:
Since this type is used only here, why don't we declare it directly inline and remove it from `commonTypes` file? if needed we can export `ExtendedDataEntry` from `commonTypes` .